### PR TITLE
frontend: add bitsurance guide

### PR DESCRIPTION
**Pull request by thisconnect**
This should answer the most common questions about Bitsurance.

### CODE REVIEW

File: frontends/web/src/routes/bitsurance/guide.tsx [hunk 1]
```diff
@@ -15,23 +15,54 @@
...
 
-// TODO
+const getFAQLink = () => {
+  switch (i18n.resolvedLanguage) {
+  case 'de':
+    return 'https://www.bitsurance.eu/bitbox/'; // TODO: double check this link
+  default:
+    return 'https://www.bitsurance.eu/en/insuring-bitcoin-on-your-bitbox-with-bitsurance-eu/';
+  }
+};
```
Comment (thisconnect) [inline frontends/web/src/routes/bitsurance/guide.tsx:29]: I am not sure if these links are correct.

**Comment by Beerosagos** [comment]
Looks good, thanks! Could you also add the guide to `dashboard.tsx`?

**Comment by thisconnect** [comment]
rebased

**Comment by thisconnect** [comment]
>  Looks good, thanks! Could you also add the guide to dashboard.tsx?

oups I missed that one. added

**Review by Beerosagos — APPROVED** [review]
tACK! Thanks

**Pull request merged**